### PR TITLE
update the copyright notice and license file to match lab standards

### PR DESCRIPTION
**Pull request by phlptp**
### Description

If merged this pull request will update the license file to match BSD standard language and update all files with the SPDX code as recommended in best practices, move the DISCLAIMER file to a NOTICE file and match institutional standards for open source releases.  

### Proposed changes

- update per file notice to include license identifier
- update the license file to match BSD-3-clause license

### CODE REVIEW

**Comment by phlptp** [comment]
obviously broke something, I will check on that the following needs to be done yet, but I wanted everyone to be able to look at the main changes
- [ ] fix cmake files
- [ ] make sure cmake files have updated notice
- [ ] update contributors file with new guidance
- [ ] update readme as per new guidance

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ A [Users guide](https://gmlc-tdc.github.io/HELICS-src/users_guide/index.html) fo
...
+Our GitHub pages provides a set of [documentation](https://gmlc-tdc.github.io/HELICS-src/index.html) including a set of introductory [examples](https://gmlc-tdc.github.io/HELICS-src/introduction/index.html), a [developers guide](https://gmlc-tdc.github.io/HELICS-src/developer-guide/index.html), complete doxygen generated [API documentation](https://gmlc-tdc.github.io/HELICS-src/doxygen/), and more.  A few more questions and answers are available on the [Wiki](https://github.com/GMLC-TDC/HELICS-src/wiki).
 
 Additionally, our initial requirements document can be found [here](https://github.com/GMLC-TDC/specification-doc/blob/master/src/specification.md), which describes a number of our early design considerations.
```
Comment (nightlark) [inline README.md]: The specification link is dead -- probably the entire line should be removed or a replacement file should be added to this repository.

**Comment by bpalmintier** [comment]
Hmm, I think we should use the latest language from the proposed IP management plan (see e-mail). This includes a LICENSE and DISCLAIMER file, along with a CONTRIBUTORS file if any third-party licensed software is used. (I will try to add this version of the LICENSE and DISCLAIMER files as a new commit on this PR.)

This also specifies the following text for all source files:

````
Copyright (c) 2017-2019,
Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
All rights reserved. See LICENSE file and DISCLAIMER file for more details.
````

Can we use this?

**Comment by phlptp** [comment]
The latest guidance from LLNL IP folks is a NOTICE file which is the same as a DISCLAIMER file and to use standard license text with no modifications other than date and copyright holders  I think the labs guidance is all happening in parallel to our discussions on this so things have shifted since the document was originally sent out.  
I am OK having it duplicated in the DISCLAIMER and NOTICE file though if you think it important

**Comment by phlptp** [comment]
I am getting the impression that the LLNL folks would not approve a modified license any longer.  Which I believe one of the PNNL folks asked about as well that the exact license text we had was somewhat non-standard from the standard BSD-3-clause text and might be rejected by some institutions even though the conditions were identical.

**Comment by phlptp** [comment]
See: https://software.llnl.gov/about/licenses/
this came out after we started sending the document around

**Comment by bpalmintier** [comment]
Oh, ugh. Sounds like we do need to circulate these latest suggestions to all labs and see if we can get a quick resolution. Using the standard text does sound like it has some other advantages, but defer to the legal/IP teams. Could you get the LLNL folks to create a revised IP management plan and send it to PNNL and PNNL?

________________________________

**Comment by bpalmintier** [comment]
Perhaps with your link?

________________________________

**Comment by bpalmintier** [comment]
Update: We have informal OK to go with the LLNL language and files. Let's revert to that approach and deploy around all of the 2.0 repos.

File: CMakeLists.txt [hunk 2]
```diff
@@ -592,223 +606,249 @@ install(TARGETS helics_base EXPORT helics-targets)
...
+        include(InstallRequiredSystemLibraries)
+    endif()
+
+    if(WIN32)
+        set(
+            CPACK_PACKAGE_ICON
+            "${CMAKE_SOURCE_DIR}\\\\docs\\\\img\\\\HELICS.ico"
+        )
+        set(CPACK_NSIS_MUI_ICON "${CMAKE_SOURCE_DIR}/docs/img/HELICS.ico")
+        set(CPACK_NSIS_INSTALL_ROOT "C:\\\\local\\\\")
```
Comment (nightlark) [inline CMakeLists.txt]: This should be `set(CPACK_NSIS_INSTALL_ROOT "C:\\\\local")` -- there is already an ending slash added by default

**Pull request merged**